app/models/chargeback/consumption_with_rollups.rb

```diff
@@ -161,10 +161,27 @@ def sub_metric_rollups(sub_metric)
     def values(metric, sub_metric = nil)
       @values ||= {}
       @values["#{metric}#{sub_metric}"] ||= begin
-        sub_metric ? sub_metric_rollups(sub_metric) : @rollup_array.collect { |x| x[ChargeableField.col_index(metric)] }.compact
+        sub_metric ? sub_metric_rollups(sub_metric) : @rollup_array.collect { |x| rollup_field(x, metric) }.compact
       end
     end
 
+    def rollup_field(rollup, metric)
+      if metric == "v_derived_cpu_total_cores_used"
+        return v_derived_cpu_total_cores_used_for(rollup)
+      end
+
+      rollup[ChargeableField.col_index(metric)]
+    end
+
+    def v_derived_cpu_total_cores_used_for(rollup)
```

Comment:
Not necessary now, but in a followup PR, this should be moved into the `MetricRollup` model